### PR TITLE
add more apps

### PR DESCRIPTION
**Pull request by Drsheppard01**

### CODE REVIEW

File: pkgs.json [hunk 1]
```diff
@@ -104,21 +124,31 @@
         "version": "0.99.54",
         "description": "A hierarchical note taking application, featuring rich text and syntax highlighting, storing data in a single XML or SQLite file.",
         "url": "https://github.com/giuspen/cherrytree/releases/download/0.99.55/CherryTree-0.99.55-x86_64.AppImage"
+    }, {
+        "name": "Chiaki",
```
Comment (blueOkiris) [inline pkgs.json:128]: Lowercase please.

File: pkgs.json [hunk 2]
```diff
@@ -136,14 +166,49 @@
         "url": "https://github.com/sqlitebrowser/sqlitebrowser/releases/download/v3.12.2/DB_Browser_for_SQLite-v3.12.2-x86_64.AppImage"
     }, {
         "name": "deltachat",
-        "version": "1.34.4",
+        "version": "1.36.1",
         "description": "Email-based instant messaging for Desktop.",
-        "url": "https://github.com/deltachat/deltachat-desktop/releases/download/v1.34.4/DeltaChat-1.34.4.AppImage"
+        "url": "https://download.delta.chat/desktop/v1.36.1/DeltaChat-1.36.1.AppImage"
+    }, {
+        "name": "digiKam",
```
Comment (blueOkiris) [inline pkgs.json:173]: Lower case

File: pkgs.json [hunk 3]
```diff
@@ -136,14 +166,49 @@
...
         "version": "0.0.26",
         "description": "All-in-one voice and text chat for gamers that's free, secure, and works on both your desktop and phone.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/discord/Discord-0.0.26-x86_64.AppImage"
+    }, {
+        "name": "drawio",
+        "version": "21.1.2",
+        "description": "Draw.io is a JavaScript, client-side editor for general diagramming and whiteboarding.",
+        "url": "https://github.com/jgraph/drawio-desktop/releases/download/v21.1.2/drawio-x86_64-21.1.2.AppImage"
+    }, {
+        "name": "DuckStation",
```
Comment (blueOkiris) [inline pkgs.json:188]: Lowercase. `duckstation` or `duck-station` works.

File: pkgs.json [hunk 4]
```diff
@@ -64,6 +74,16 @@
         "version": "2.8.1",
         "description": "Avidemux is a free video editor designed for simple cutting, filtering and encoding tasks.",
         "url": "https://www.fosshub.com/Avidemux.html?dwl=avidemux_2.8.1.appImage"    
+    }, {
+        "name": "axolotl",
+        "version": "1.6.0",
+        "description": "A Signal compatible cross plattform client written in Go and VueJS.",
+        "url": "https://github.com/nanu-c/axolotl/releases/download/v1.6.0/Axolotl-x86_64.AppImage"
+    }, {
+        "name": "Beekeeper-Studio",
```
Comment (blueOkiris) [inline pkgs.json:83]: Lowercase please

File: pkgs.json [hunk 5]
```diff
@@ -1,5 +1,10 @@
 [
     {
+        "name": "86Box",
```
Comment (blueOkiris) [inline pkgs.json:3]: Please keep package names lower case for consistency, so `86box` or `86-box`
Afterall, it's not the name of the app itself, but just what aip-man uses when doing `install` and such.

It's pretty standard to do lowercase for package names in package managers, and while aip-man could've been different, most of the other packages are already lowercase.

File: pkgs.json [hunk 6]
```diff
@@ -46,9 +51,14 @@
         "url": "https://github.com/AppImage/AppImageKit/releases/download/13/appimagetool-x86_64.AppImage"
     }, {
         "name": "arduino-ide",
-        "version": "2.0.4",
+        "version": "2.1.0",
         "description": "Arduino is a development environment that implements the Processing/Wiring language.",
-        "url": "https://github.com/arduino/arduino-ide/releases/download/2.0.4/arduino-ide_2.0.4_Linux_64bit.AppImage"
+        "url": "https://github.com/arduino/arduino-ide/releases/download/2.1.0/arduino-ide_2.1.0_Linux_64bit.AppImage"
+    }, {
+        "name": "ArmCord",
```
Comment (blueOkiris) [inline pkgs.json:58]: Lowercase to `armcord` or `arm-cord` please

File: pkgs.json [hunk 7]
```diff
@@ -194,21 +259,41 @@
         "version": "5.9.2",
         "description": "Franz is a free messaging app for services like WhatsApp, Slack, Messenger and many more.",
         "url": "https://github.com/meetfranz/franz/releases/download/v5.9.2/Franz-5.9.2.AppImage"
+    }, {
+        "name": "Frappe-Books",
```
Comment (blueOkiris) [inline pkgs.json:263]: Lowercase please

File: pkgs.json [hunk 8]
```diff
@@ -194,21 +259,41 @@
...
+        "version": "2.18.0",
+        "description": "Gaphor is the simple modeling tool.",
         "url": "https://github.com/gaphor/gaphor/releases/download/2.18.0/Gaphor-2.18.0-x86_64.AppImage"
     }, {
         "name": "gimp",
         "version": "2.10.13",
         "description": "Create images and edit photographs.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/gimp/GIMP_AppImage-git-2.10.13-20191010-withplugins-x86_64.AppImage"
+    }, {
+        "name": "GitHubDesktop",
```
Comment (blueOkiris) [inline pkgs.json:288]: Lowercase please

File: pkgs.json [hunk 9]
```diff
@@ -219,43 +304,93 @@
...
+        "version": "22.12.3",
+        "description": "Kate is a feature-packed Text Editor.",
+        "url": "https://binary-factory.kde.org/job/Kate_Release_appimage-centos7/lastSuccessfulBuild/artifact/kate-22.12.3-837-linux-gcc-x86_64.AppImage"
     }, {
         "name": "kdenlive",
         "version": "22.12.3",
         "description": "Kdenlive is a non linear video editor.",
         "url": "https://download.kde.org/stable/kdenlive/22.12/linux/kdenlive-22.12.3-x86_64.AppImage"
+    }, {
+        "name": "KeePassXC",
```
Comment (blueOkiris) [inline pkgs.json:353]: Lowercase please

File: pkgs.json [hunk 10]
```diff
@@ -194,21 +259,41 @@
...
+        "version": "0.12.0",
+        "description": "Free Desktop book-keeping software for small businesses and freelancers.",
+        "url": "https://github.com/frappe/books/releases/download/v0.12.0/Frappe-Books-0.12.0.AppImage"
     }, {
         "name": "freecad",
         "version": "0.20.2",
         "description": "FreeCAD is a parametric 3D modeler.",
         "url": "https://github.com/FreeCAD/FreeCAD/releases/download/0.20.2/FreeCAD_0.20.2-2022-12-27-conda-Linux-x86_64-py310.AppImage"
+    }, {
+        "name": "GameHub",
```
Comment (blueOkiris) [inline pkgs.json:273]: Lowercase please

File: pkgs.json [hunk 11]
```diff
@@ -219,43 +304,93 @@
...
+        "description": "Kopia is a fast and secure open-source backup/restore tool.",
+        "url": "https://github.com/kopia/kopia/releases/download/v0.13.0-rc1/KopiaUI-0.13.0-rc1.AppImage"
     }, {
         "name": "krita",
         "version": "5.1.5",
         "description": "Krita is a professional FREE and open source painting program. It is made by artists that want to see affordable art tools for everyone.",
         "url": "https://download.kde.org/stable/krita/5.1.5/krita-5.1.5-x86_64.appimage"
     }, {
-        "name": "kuro",
+        "name": "Kuro",
```
Comment (blueOkiris) [inline pkgs.json:368]: Lowercase please

File: pkgs.json [hunk 12]
```diff
@@ -219,43 +304,93 @@
...
         "description": "An elegant Microsoft ToDo desktop client for Linux (a fork of Ao).",
         "url": "https://github.com/davidsmorais/kuro/releases/download/v9.0.0/Kuro-9.0.0.AppImage"
     }, {
-        "name": "librecad",
+        "name": "lens",
+        "version": "2023.4.60821",
+        "description": "Lens is the way the world runs Kubernetes.",
+        "url": "https://api.k8slens.dev/binaries/Lens-2023.4.60821-latest.x86_64.AppImage"
+    }, {
+        "name": "LeoCAD",
```
Comment (blueOkiris) [inline pkgs.json:378]: Lowercase please

File: pkgs.json [hunk 13]
```diff
@@ -219,43 +304,93 @@
...
+        "version": "23.03",
+        "description": "Design virtual models you can build with LEGO® bricks.",
+        "url": "https://github.com/leozide/leocad/releases/download/v23.03/LeoCAD-Linux-23.03-x86_64.AppImage"
+    }, {
+        "name": "LibreCAD",
```
Comment (blueOkiris) [inline pkgs.json:383]: Lowercase please

File: pkgs.json [hunk 14]
```diff
@@ -219,43 +304,93 @@
...
         "description": "LibreCAD is a 2D CAD drawing tool based on the community edition of QCAD. LibreCAD uses the cross-platform framework Qt, which means it works with most operating systems.",
         "url": "https://github.com/LibreCAD/LibreCAD/releases/download/2.2.0/LibreCAD-2.2.0-x86_64.AppImage"
     }, {
-        "name": "libreoffice-fresh-basic",
+        "name": "librepcb",
+        "version": "0.1.7",
+        "description": "LibrePCB is a free, cross-platform, easy-to-use electronic design automation suite to draw schematics and design printed circuit boards – for makers, students and professionals, from beginners to experts.",
+        "url": "https://download.librepcb.org/releases/0.1.7/librepcb-0.1.7-linux-x86_64.AppImage"
+    }, {
+        "name": "LibreOffice-fresh-basic",
```
Comment (blueOkiris) [inline pkgs.json:393]: Lowercase please

File: pkgs.json [hunk 15]
```diff
@@ -266,40 +401,80 @@
         "url": "https://github.com/LMMS/lmms/releases/download/v1.3.0-alpha.1/lmms-1.3.0-alpha.1.102%2Bg89fc6c9-linux-x86_64.AppImage"
     }, {
         "name": "logseq",
-        "version": "0.9.1",
+        "version": "0.9.3",
         "description": "A privacy-first, open-source platform for knowledge management and collaboration",
-        "url": "https://github.com/logseq/logseq/releases/download/0.9.1/Logseq-linux-x64-0.9.1.AppImage"
+        "url": "https://github.com/logseq/logseq/releases/download/0.9.3/Logseq-linux-x64-0.9.3.AppImage"
+    }, {
+        "name": "Login_Manager_Settings",
```
Comment (blueOkiris) [inline pkgs.json:408]: Lowercase please, and use hyphen over underscore, so `login-manager-settings`

File: pkgs.json [hunk 16]
```diff
@@ -266,40 +401,80 @@
...
+        "version": "0.17.1",
+        "description": "Simple and elegant markdown editor.",
+        "url": "https://github.com/marktext/marktext/releases/download/v0.17.1/marktext-x86_64.AppImage"
     }, {
         "name": "mendeley",
         "version": "2.86.0",
         "description": "Mendeley Reference Manager is a free web and desktop reference management application",
         "url": "https://static.mendeley.com/bin/desktop/mendeley-reference-manager-2.86.0-x86_64.AppImage"
+    }, {
+        "name": "Mochi",
```
Comment (blueOkiris) [inline pkgs.json:433]: Lowercase please

File: pkgs.json [hunk 17]
```diff
@@ -266,40 +401,80 @@
...
+        "version": "0.13.1",
+        "description": "A simple, clean and cross-platform music player.",
+        "url": "https://github.com/martpie/museeks/releases/download/0.13.1/museeks-x86_64.AppImage"
     }, {
         "name": "musescore",
         "version": "4.0.2",
         "description": "Create, playback, and print sheet music for free. MuseScore is cross-platform, multi-lingual, open source music notation software.",
         "url": "https://github.com/musescore/MuseScore/releases/download/v4.0.2/MuseScore-4.0.2.230651545-x86_64.AppImage"
     }, {
+        "name": "MQTTX",
```
Comment (blueOkiris) [inline pkgs.json:468]: Lowercase please

File: pkgs.json [hunk 18]
```diff
@@ -266,40 +401,80 @@
...
+        "version": "1.9.1",
+        "description": "Powerful cross-platform MQTT 5.0 Desktop, CLI, and WebSocket client tools.",
+        "url": "https://github.com/emqx/MQTTX/releases/download/v1.9.1/MQTTX-1.9.1.AppImage"
+    }, {
+        "name": "MyPaint",
```
Comment (blueOkiris) [inline pkgs.json:473]: Lowercase please

File: pkgs.json [hunk 19]
```diff
@@ -359,11 +554,36 @@
         "version": "3.1.0",
         "description": "OpenShot Video Editor is an award-winning free and open-source video editor for Linux, Mac, and Windows, and is dedicated to delivering high quality video editing and animation solutions to the world.",
         "url": "https://github.com/OpenShot/openshot-qt/releases/download/v3.1.0/OpenShot-v3.1.0-x86_64.AppImage"
+     }, {
+        "name": "ownCloud",
```
Comment (blueOkiris) [inline pkgs.json:558]: Lowercase please

File: pkgs.json [hunk 20]
```diff
@@ -359,11 +554,36 @@
...
+        "version": "1.3.0",
+        "description": "Offline password manager with cross-platform synchronization.",
+        "url": "https://github.com/GlitterWare/Passy/releases/download/v1.3.0/Passy-v1.3.0-Linux-x86_64.AppImage"
     }, {
         "name": "pencil2d",
         "version": "0.6.6",
         "description": "Pencil2D is an easy, intuitive tool to make 2D hand-drawn animations. Pencil2D is open source and cross-platform.",
         "url": "https://github.com/pencil2d/pencil/releases/download/v0.6.6/pencil2d-linux-amd64-0.6.6.AppImage"
+    }, {
+        "name": "PhotoQt",
```
Comment (blueOkiris) [inline pkgs.json:583]: Lowercase please

File: pkgs.json [hunk 21]
```diff
@@ -374,11 +594,21 @@
         "version": "1807",
         "description": "Generate AppImages from existing binary packages.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/pkg2appimage/pkg2appimage-1807-x86_64.AppImage"
+    }, {
+        "name": "Play!",
```
Comment (blueOkiris) [inline pkgs.json:598]: Lowercase please and no ! (people would have to escape it to install)

File: pkgs.json [hunk 22]
```diff
@@ -389,21 +619,66 @@
...
         "version": "1.103.0",
         "description": "A Community-led Hyper-Hackable Text Editor.",
         "url": "https://github.com/pulsar-edit/pulsar/releases/download/v1.103.0/Linux.1.103.0.AppImage"
     }, {
+        "name": "qbittorrent",
+        "version": "4.5.2",
+        "description": "The qBittorrent project aims to provide an open-source software alternative to µTorrent.",
+        "url": "https://www.fosshub.com/qBittorrent.html?dwl=qbittorrent-4.5.2_lt20_x86_64.AppImage"
+    }, {
+        "name": "QElectrotech",
```
Comment (blueOkiris) [inline pkgs.json:638]: Lowercase please

File: pkgs.json [hunk 23]
```diff
@@ -389,21 +619,66 @@
...
+        "version": "0.100.0",
+        "description": "QElectroTech is a free software to create electric diagrams.",
+        "url": "https://download.tuxfamily.org/qet/builds/AppImage/QElectroTech_0.100.0-r7828-x86_64.AppImage"
+    }, {
+        "name": "QOwnNotes",
```
Comment (blueOkiris) [inline pkgs.json:643]: Lowercase please

File: pkgs.json [hunk 24]
```diff
@@ -389,21 +619,66 @@
...
+        "version": "0.9.33",
+        "description": "Qtractor is an Audio/MIDI multi-track sequencer application.",
+        "url": "https://sourceforge.net/projects/qtractor/files/qtractor/0.9.33/qtractor-0.9.33-76.1.x86_64.AppImage"
+    }, {    
         "name": "quaternion",
         "version": "0.0.96.1",
         "description": "A Qt5-based IM client for Matrix.",
         "url": "https://github.com/quotient-im/Quaternion/releases/download/0.0.96-beta1/quaternion-0.0.96-beta1.AppImage"
     }, {
+        "name": "qView",
```
Comment (blueOkiris) [inline pkgs.json:663]: Lowercase please

File: pkgs.json [hunk 25]
```diff
@@ -389,21 +619,66 @@
...
+        "version": "5.0",
+        "description": "qView is an image viewer designed with minimalism and usability in mind.",
+        "url": "https://github.com/jurplel/qView/releases/download/5.0/qView-5.0-x86_64.AppImage"
+    }, {
+        "name": "Rare",
```
Comment (blueOkiris) [inline pkgs.json:668]: Lowercase please

File: pkgs.json [hunk 26]
```diff
@@ -389,21 +619,66 @@
...
+        "version": "1.9.5",
+        "description": "Rare is a graphical interface for Legendary, a command line alternative to Epic Games launcher, based on PyQt5.",
+        "url": "https://github.com/Dummerle/Rare/releases/download/1.9.5/Rare-1.9.5.AppImage"
+    }, {
+        "name": "RawTherapee",
```
Comment (blueOkiris) [inline pkgs.json:673]: Lowercase please

File: pkgs.json [hunk 27]
```diff
@@ -414,16 +689,31 @@
         "version": "3.0.15-release",
         "description": "Anonymous peer-to-peer instant messaging.",
         "url": "https://github.com/blueprint-freespeech/ricochet-refresh/releases/download/v3.0.15-release/Ricochet_Refresh-156ddee-x86_64.AppImage"
+    }, {
+        "name": "RPCS3",
```
Comment (blueOkiris) [inline pkgs.json:693]: Lowercase please

File: pkgs.json [hunk 28]
```diff
@@ -437,13 +727,33 @@
...
+        "version": "1.3.1",
+        "description": "sleek is an open-source (FOSS) todo manager based on the todo.txt syntax.",
+        "url": "https://github.com/ransome1/sleek/releases/download/v1.3.1/sleek-1.3.1.AppImage"
     }, {
         "name": "slippi-launcher",
         "version": "2.8.1",
         "description": "The Slippi Launcher is the hub for everything Slippi. It downloads and auto-updates Slippi Dolphin along with providing access to news, replays, mirroring, broadcasting, and more.",
         "url": "https://github.com/project-slippi/slippi-launcher/releases/download/v2.8.1/Slippi-Launcher-2.8.1-x86_64.AppImage"
+    }, {
+        "name": "SMPlayer",
```
Comment (blueOkiris) [inline pkgs.json:748]: Lowercase please

File: pkgs.json [hunk 29]
```diff
@@ -452,13 +762,38 @@
...
         "description": "End-to-end encrypted notes app.",
-        "url": "https://github.com/standardnotes/app/releases/download/%40standardnotes%2Fdesktop%403.150.48/standard-notes-3.150.48-linux-x86_64.AppImage"
+        "url": "https://github.com/standardnotes/app/releases/download/%40standardnotes%2Fdesktop%403.150.54/standard-notes-3.150.54-linux-x86_64.AppImage"
+    }, {
+        "name": "starc",
+        "version": "0.4.5",
+        "description": "Story Architect is a project created by the authors of an open source screenwriting tool.",
+        "url": "https://github.com/story-apps/starc/releases/download/v0.4.5/starc-setup.AppImage"
+    }, {
+        "name": "Storyboarder",
```
Comment (blueOkiris) [inline pkgs.json:783]: Lowercase please

File: pkgs.json [hunk 30]
```diff
@@ -452,13 +762,38 @@
...
+        "version": "2.1.0",
+        "description": "Storyboarder makes it easy to visualize a story as fast you can draw stick figures.",
+        "url": "https://github.com/wonderunit/storyboarder/releases/download/v2.1.0/Storyboarder-2.1.0-linux-x86_64.AppImage"
+    }, {
+        "name": "Syncplay",
```
Comment (blueOkiris) [inline pkgs.json:788]: Lowercase please

File: pkgs.json [hunk 31]
```diff
@@ -467,42 +802,132 @@
...
+        "version": "2.2.0",
+        "description": "Thorium Reader is an easy to use EPUB reading application.",
+        "url": "https://github.com/edrlab/thorium-reader/releases/download/v2.2.0/Thorium-2.2.0.AppImage"
     }, {
         "name": "thunar",
         "version": "4.18.3",
         "description": "Xfce4 file manager.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/thunar/thunar-4.18.3-x86_64.AppImage"
+    }, {
+        "name": "Tropy",
```
Comment (blueOkiris) [inline pkgs.json:818]: Lowercase please

File: pkgs.json [hunk 32]
```diff
@@ -467,42 +802,132 @@
...
+        "version": "1.3.3",
+        "description": "Twake is a secure open source collaboration platform to improve organizational productivity.",
+        "url": "https://github.com/linagora/Twake-Desktop/releases/download/v1.3.3/Setup_Twake_lastest.AppImage"
+    }, {
+        "name": "upscayl",
+        "version": "2.0.1",
+        "description": "Upscayl - Free and Open Source AI Image Upscaler for.",
+        "url": "https://github.com/upscayl/upscayl/releases/download/v2.0.1/upscayl-2.0.1-linux.AppImage"
+    }, {
+        "name": "Vieb",
```
Comment (blueOkiris) [inline pkgs.json:838]: Lowercase please

File: pkgs.json [hunk 33]
```diff
@@ -467,42 +802,132 @@
...
+        "version": "0.12.0",
+        "description": "Vup is an open-source end-to-end-encrypted cloud storage app powered by the S5 Network.",
+        "url": "https://github.com/vup-app/vup/releases/download/v0.12.0/Vup-0.12.0-BETA-Linux.AppImage"
     }, {
         "name": "wavebox",
         "version": "10.112.6-2",
         "description": "Wavebox, the revolutionary and feature-rich Chromium browser that's built for productive working across Google Workspaces, Microsoft Teams, ClickUp, Monday, Atlassian, Asana, AirTable, Slack, and every other web app you use to get work done.",
         "url": "https://download.wavebox.app/stable/linux/appimage/Wavebox_10.112.6-2_x86_64.AppImage"
+    }, {
+        "name": "WebCord",
```
Comment (blueOkiris) [inline pkgs.json:868]: Lowercase please

File: pkgs.json [hunk 34]
```diff
@@ -467,42 +802,132 @@
...
+        "version": "2.0.0",
+        "description": "WeekToDo is a Free and Open Source Minimalist Weekly Planner and To Do list App focused on privacy",
+        "url": "https://github.com/manuelernestog/weektodo/releases/download/v2.0.0/WeekToDo-2.0.0.AppImage"
     }, {
         "name": "wezterm",
         "version": "20230408-112425-69ae8472",
         "description": "A GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust.",
         "url": "https://github.com/wez/wezterm/releases/download/20230408-112425-69ae8472/WezTerm-20230408-112425-69ae8472-Ubuntu20.04.AppImage"
+    }, {
+        "name": "Whalebird",
```
Comment (blueOkiris) [inline pkgs.json:883]: Lowercase please

File: pkgs.json [hunk 35]
```diff
@@ -467,42 +802,132 @@
...
-        "description": "High Appearance Third-Party Netease CloudMusic Player",
+        "description": "High Appearance Third-Party Netease CloudMusic Player.",
         "url": "https://github.com/qier222/YesPlayMusic/releases/download/v0.4.7/YesPlayMusic-0.4.7.AppImage"
+    }, {
+        "name": "yuzu-emu",
+        "version": "mainline-0-1403",
+        "description": "Yuzu is open-source, Nintendo Switch emulator.",
+        "url": "https://github.com/yuzu-emu/yuzu-mainline/releases/download/mainline-0-1403/yuzu-mainline-20230414-33cca53c7.AppImage"
+    }, {
+        "name": "Zettlr",
```
Comment (blueOkiris) [inline pkgs.json:918]: Lowercase please

File: pkgs.json [hunk 36]
```diff
@@ -467,42 +802,132 @@
...
+        "description": "A Markdown Editor for the 21st century.",
+        "url": "https://github.com/Zettlr/Zettlr/releases/download/v2.3.0/Zettlr-2.3.0-x86_64.AppImage"
     }, {
         "name": "zoom",
         "version": "5.13.7.683",
-        "description": "Zoom Video Conference",
+        "description": "Zoom Video Conference.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/zoom/Zoom-5.13.7.683.glibc2.17-x86_64.AppImage"
+    }, {
+        "name": "Zulip",
```
Comment (blueOkiris) [inline pkgs.json:928]: Lowercase please

**Review by blueOkiris — CHANGES_REQUESTED** [review]
It's mostly good, but I have one nitpick.

I'd like package names to be consistent and match what people expect of a package manager which is typically:
- Lowercase
- Hyphenated instead of underscored
- No special characters

You've added quite a few with PascalCase, and I'd like them to match the existing package-names. I tried to mark all of them, so it's easier for you to find.

File: pkgs.json (hunk 15 above)
Comment (Drsheppard01) [reply pkgs.json]: renamed in gdm-settings and ordered

File: pkgs.json [hunk 37]
```diff
@@ -194,21 +259,46 @@
...
+        "version": "3.1",
+        "description": "A settings app for GNOME's Login Manager, GDM.",
+        "url": "https://github.com/gdm-settings/gdm-settings/releases/download/v3.1/Login_Manager_Settings.AppImage"    
     }, {
         "name": "gimp",
         "version": "2.10.13",
         "description": "Create images and edit photographs.",
         "url": "https://github.com/blueOkiris/aip-man-pkgs/raw/gimp/GIMP_AppImage-git-2.10.13-20191010-withplugins-x86_64.AppImage"
+    }, {
+        "name": "gitHubdesktop",
```
Comment (blueOkiris) [inline pkgs.json:293]: Missed one of the lowercases

**Review by blueOkiris — CHANGES_REQUESTED** [review]

**Review by blueOkiris — APPROVED** [review]
Looks good. Thanks for adding

**Pull request merged**